app/src/main/java/org/sopt/dosopttemplate/presentation/type/ScrollableView.kt

```diff
@@ -0,0 +1,5 @@
+package org.sopt.dosopttemplate.presentation.type
+
+interface ScrollableView {
+    fun scrollToTop()
+}
```

Comment:
```suggestion
fun interface ScrollableView {
    fun scrollToTop()
}
```
[SAM](https://kotlinlang.org/docs/fun-interfaces.html)

Reply:
미쳐따 ㅋ.ㅋ 감사합니닷